redis-sink/src/main/resources/application.yml

```diff
@@ -0,0 +1,12 @@
+server:
+  port: 8081
+spring:
+  cloud:
+    stream:
+      bindings:
+        input: testtock
```

Comment:
we need to avoid falling into the trap of thinking that spring-cloud-stream apps will only be used in spring-cloud-data *streams*... instead they should be considered valid as apps in their own right